docs/Projects/TAD/index.md

```diff
@@ -0,0 +1,26 @@
+# TAD
+
+TAD is the acronym for **T**ransparency of **A**lgorithmic **D**ecision making, the goal is to make algorithmic systems more
+transparent by generating standardized reports on the algorithmic system for both it's technical element as well as the
```

Comment:
Also we should maybe add some explenations about the used terminology. For example: "An algorithmic system is a system consisting out of one or more ML models".

Reply:
I just added a reference to the definition of an algorithm of the algoritme register, would this suffice or would you like to descripe also the algorithm system better?